README.md

```diff
@@ -1,64 +1,58 @@
+<div align="center">
+
 # @skyra/editable-commands
 
+**A framework agnostic library for editable commands**
+
 [![GitHub](https://img.shields.io/github/license/skyra-project/editable-commands)](https://github.com/skyra-project/editable-commands/blob/main/LICENSE.md)
 [![Depfu](https://badges.depfu.com/badges/e367f2c68b857253ca23e1e8d73d1e14/count.svg)](https://depfu.com/github/skyra-project/editable-commands?project_id=14147)
 
 [![npm](https://img.shields.io/npm/v/@skyra/editable-commands?color=crimson&label=NPM&logo=npm&style=flat-square)](https://www.npmjs.com/package/@skyra/editable-commands)
 ![npm bundle size minified (scoped)](https://img.shields.io/bundlephobia/min/@skyra/editable-commands?label=minified&logo=webpack)
 ![npm bundle size minzipped (scoped)](https://img.shields.io/bundlephobia/minzip/@skyra/editable-commands?label=minified&logo=webpack)
 
-**Table of Contents**
+[![Support Server](https://discord.com/api/guilds/254360814063058944/embed.png?style=banner2)](https://join.skyra.pw)
+
+</div>
 
--   [@skyra/editable-commands](#skyraeditable-commands)
-    -   [About](#about)
-    -   [Installation and Usage](#installation-and-usage)
-        -   [Package managers](#package-managers)
-            -   [Usage](#usage)
-    -   [Meta](#meta)
-        -   [License](#license)
-        -   [Contributing](#contributing)
-        -   [Buy us some doughnuts](#buy-us-some-doughnuts)
-        -   [Contributors ✨](#contributors-%E2%9C%A8)
+---
 
-## About
+## Features
 
 -   @skyra/editable-commands is a framework agnostic implementation of editable commands for discord.js v13.
+-   Supports CommonJS and ES Module.
 
 ## Installation and Usage
 
-### Package managers
+You can use the following command to install this package, or replace `npm install` with your package manager of choice.
 
-```bash
-yarn add @skyra/editable-commands
-# or npm install @skyra/editable-commands
+```sh
+npm install @skyra/editable-commands
 ```
 
-#### Usage
+## Usage
+
+**For CommonJS**:
 
 ```js
-require('@skyra/editable-commands');
-```
+const { send } = require('@skyra/editable-commands');
 
-```ts
-import '@skyra/editable-commands';
+client.on('messageCreate', message => {
+	send(message, 'This is my reply!');
+});
```

Comment:
Perhaps also include a `messageUpdate` event? Otherwise this never edits.

Also... this doesn't pass our linter:

```suggestion
client.on('messageCreate', (message) => {
	send(message, 'This is my reply!');
});
```